assessment.rb

```diff
@@ -2,11 +2,26 @@
 # Assessment answers to:
 
 # question 2:
-
+person = Licensee.create surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 132, dob: '1943-09-26', eye_color: 'Brown', hair_color: 'Black'
 # question 3:
-
+Licensee.transaction do
+  CSV.foreach(Rails.root + 'data/licensees.csv', header: true) do |licensee_row|
+    person = licensee_row.to_hash
+    next if Licensee.exists? person
+    Licensee.create! person
+  end
+end
 # question 4:
-
+Licensee.where(eye_color = 'Hazel').where(hair_color = 'Brown' OR hair_color = 'Black')
```

Comment:
Not quite syntactically correct. There are couple of different ways to do this one - here's one.
`Licensee.where(eye_color: 'Hazel', hair_color: ['Brown', 'Black'])`

Can you figure out any others?